src/common/feature/chamber/chamber.hpp

```diff
@@ -4,7 +4,7 @@
 
 #include <option/xl_enclosure_support.h>
 #include <option/has_xbuddy_extension.h>
-#include <common/temperature.hpp>
+#include <../../temperature.hpp>
```

Comment:
Not acceptable. If the chamber hpp does not work in marlin, include directories should be fixed.